build_package.py

```diff
@@ -879,6 +891,26 @@ def build_base_files(base_configs="config"):
     shutil.copy(
         make_path(agent_source_root, "VERSION"), os.path.join("scalyr_agent", "VERSION")
     )
+
+    # create copy of the agent_main.py with python3 shebang.
+    agent_main_path = os.path.join(agent_source_root, "scalyr_agent", "agent_main.py")
+    agent_main_py3_path = os.path.join("scalyr_agent", "agent_main_py3.py")
+    replace_shebang(
+        agent_main_path,
+        agent_main_py3_path,
+        "#!/usr/bin/env python3"
+    )
+
+    # create copy of the config_main.py with python3 shebang.
+    config_main_path = os.path.join(agent_source_root, "scalyr_agent", "config_main.py")
+    replace_shebang(
+        config_main_path,
+        os.path.join("scalyr_agent", "config_main_py3.py"),
+        "#!/usr/bin/env python3"
+    )
+
+    os.chmod(agent_main_py3_path, stat.S_IREAD | stat.S_IEXEC)
```

Comment:
Yeah, ideally we would just copy the permissions of the existing ``agent_main.py`` file.

Reply:
yup, thanks for the good advice.